internal/types/types.go

```diff
@@ -381,6 +381,7 @@ type IndexCoord interface {
 	CreateIndex(ctx context.Context, req *indexpb.CreateIndexRequest) (*commonpb.Status, error)
 
 	// GetIndexState gets the index state of the index name in the request from Proxy.
+	// Deprecated: use DescribeIndex instead
```

Comment:
One quick question: should we forward these request to newer `DescribeIndex`, for compatibility issues?

Reply:
I think it's no need, because this rpc will be removed in the near future.